tml/org.testeditor.tml.model/model/tml.xcore

```diff
@@ -72,13 +72,13 @@ class AEComparison extends BinaryAssertionExpression {
 	contains Comparator comparator
 }
 
-class AENullCheck extends AssertionExpression {
+class AENullOrBoolCheck extends AssertionExpression {
 	contains AEVariableReference varReference
 	boolean negated = "false"
 }
 
 class AEVariableReference extends AssertionExpression {
-	String name
+	refers TestStepWithAssignment testStepWithAssignment
```

Comment:
I'm not happy with this long reference name either but since I want to reference the assignment (variable) I would be even unhappier to call it testStep. I would rather split the TestStepWithAssignment into an AssignmentVariable and TestStep and then reference the AssignmentVariable ...